images/linux/scripts/installers/java-tools.sh

```diff
@@ -45,8 +45,14 @@ enableRepositories() {
     fi
 
     # Add Addoptium PPA
-    wget -qO - "https://packages.adoptium.net/artifactory/api/gpg/key/public" | apt-key add -
-    add-apt-repository --yes https://packages.adoptium.net/artifactory/deb/
+    if isUbuntu22; then
```

Comment:
Why do we need this condition if gpg --dearmor works for Ubuntu18+?

Reply:
@al-cheb what is the point to dearmor it on ubuntu18+ if apt-key does it here?

Reply:
@al-cheb or do you mean we must switch off apt-key generally? then it makes sense to me